src/backend/core/admin.py

```diff
@@ -108,4 +108,5 @@ class TeamAdmin(admin.ModelAdmin):
         "created_at",
         "updated_at",
     )
+    prepopulated_fields = {"slug": ("name",)} 
```

Comment:
Should we prevent editing this field, as initially we introduced a slug, to make sure all teams' name would be unique?